views/homework/edit.hbs

```diff
@@ -54,6 +54,7 @@
 							$('#lessonPicker').append('<option value="' + r.lessons.data[i]._id + '">' + r.lessons.data[i].name + '</option>');
 						}
 						$('#lessonPicker').prop('disabled', false);
+						$('#lessonPicker').trigger("chosen:updated");
```

Comment:
you should update the picker also for the case when there were no tasks found.

Reply:
I would put it to line 60 and also probably to line 64. Please check whether it is required.

Reply:
done ;)